### PR TITLE
Modify rule S1144: LaYC format

### PR DESCRIPTION
**Pull request by costin-zaharia-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S1144/csharp/rule.adoc [hunk 1]
```diff
@@ -34,32 +53,11 @@ public class Foo
...
 
-* Remove this unused [private|internal] member.
-* Remove this unused private property [getter|setter].
+=== Documentation
 
+* https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers[Access Modifiers (C# Programming Guide)]
 
-'''
-== Comments And Links
-(visible only on this page)
```
Comment (csaba-sagi-sonarsource) [inline rules/S1144/csharp/rule.adoc:62]: Extra empty line that can be removed

**Review by csaba-sagi-sonarsource — CHANGES_REQUESTED** [review]
The C# changes looks good, the only reason it is not merged is that we agreed to remove the targeted languages, so please remove abap, javascript, vb6 and vbnet.

**Comment by costin-zaharia-sonarsource** [comment]
> The C# changes looks good, the only reason it is not merged is that we agreed to remove the targeted languages, so please remove abap, javascript, vb6 and vbnet.

Isn't this handled in a separate PR?

**Review by csaba-sagi-sonarsource — APPROVED** [review]
Indeed they will be removed in [this](https://github.com/SonarSource/rspec/pull/2145) PR

**Pull request merged**